openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/components/TestCaseForm.tsx

```diff
@@ -285,12 +286,18 @@ const TestCaseForm: React.FC<TestCaseFormProps> = ({
                 field: t('label.column'),
               })}`,
             },
-          ]}>
+          ]}
+          tooltip={testDefinition?.description}>
           <Select
             data-testid="column"
             placeholder={t('label.please-select-entity', {
               entity: t('label.column-lowercase'),
-            })}>
+            })}
+            onChange={(value) => {
+              setTestDefinition(
+                  testDefinitions.find((item) => item.fullyQualifiedName === value) as TestDefinition
+              );
+            }}>
```

Comment:
it would be great if we pull out the inline function to separate function in the code with below changes
```suggestion
            onChange={(value) => {
              setTestDefinition(
                  testDefinitions.find((item) => item.fullyQualifiedName === value)
              );
            }}>
```